City-of-Helsinki/drupal-helfi-kymp
# UHF-5896 Enable news list module

# Enable news list paragraph (https://helsinkisolutionoffice.atlassian.net/browse/UHF-5896)

## What was done
* Enabled the news feed module that provides the news list paragraph.
* Updated translations for the site as well.

## How to install
* Checkout this branch on your latest dev KYMP installation
* `make drush-cim && make drush-cr && make drush-cim`

## How to test
* Go to edit any landing page and make sure that you are able to add news list paragraph to the content. Add one without any tags or other configuration.
* To make some news appear on local you need to add the following line to local.settings.php:
* `$config['helfi_news_feed.settings']['source_environment'] = 'test';`
* Run `make drush-cr` and you should see a list of news items from testing server. The list is not styled and it is done in a separate task.

## Designers review

One of the checkboxes below needs to be checked (`[x]`)

* [ ] This PR has been visually reviewed by a designer (Designer)
